### PR TITLE
[Inductor] Fix test_conv2d_unary_cpu_cpp_wrapper failure

### PR DESCRIPTION
**Pull request by desertfire**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #137158

Summary: test_conv2d_unary_cpu_cpp_wrapper is failing on ciflow/slow because of mis-handling of inf. This PR fixes that.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @chauhang

### CODE REVIEW

File: torch/_inductor/codegen/cpp_wrapper_cpu.py [hunk 1]
```diff
@@ -2211,7 +2222,7 @@ def generate_py_arg_inner(lines, raw_arg, arg_type):
                 )
                 return f"PyLong_FromLongLong({self.expr_printer(expr)})"
             elif isinstance(arg_type, torch.FloatType):
```
Comment (henrylhtsang) [inline torch/_inductor/codegen/cpp_wrapper_cpu.py:2224]: @desertfire would we get assertion error because arg_type is torch.FloatType instead of float?

**Review by chenyang78 — APPROVED** [review]

File: torch/_inductor/codegen/cpp_wrapper_cpu.py (hunk 1 above)
Comment (desertfire) [reply torch/_inductor/codegen/cpp_wrapper_cpu.py:2224]: `torch.FloatType` is the type info from an op schema, `val` should still be float here.

**Pull request closed without merging**